R/cluster_rs.R

```diff
@@ -37,20 +38,25 @@ cluster_rs <- function(clusters = NULL,
   if (!is.null(prob_unit)) {
     prob_unit <- tapply(prob_unit, INDEX = clusters, FUN = unique)
   }
+  
   if (!is.null(n_unit)) {
-    n <- unique(n_unit)
+    n_unit <- tapply(n_unit, INDEX = clusters, FUN = unique)
   }
   
   if (simple) {
-    S_clust <- simple_rs(N = n_clust, prob = prob, prob_unit = prob_unit)
+    S_clust <-
```

Comment:
`S_clust <- simple_rs(`  all on one line (many changes like this should get fixed).